### PR TITLE
wofi-emoji: 2021-05-24 -> 2022-08-19

### PR DESCRIPTION
**Pull request by algorithmiker**
###### Description of changes

No upstream changelog, GitHub diff of the previous and this commit: https://github.com/dln/wofi-emoji/compare/bfe35c1..HEAD
Added missing dependencies; without those the program would fail at runtime (it did previously)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
I couldn't get these to run, my low-end hardware ran out of memory :( (or something related to the CPU, not sure)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by riceicetea** [comment]
Pinging previous maintainer: @ymarkus

File: pkgs/applications/misc/wofi-emoji/default.nix [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
     owner = "dln";
     repo = pname;
-    rev = "bfe35c1198667489023109f6843217b968a35183";
-    sha256 = "sha256-wMIjTUCVn4uF0cpBkPfs76NRvwS0WhGGJRy9vvtmVWQ=";
+    rev = "c5ecb4f0f164aedb046f52b5eacac889609c8522";
+    sha256 = "1wq276bhf9x24ds13b2dwa69cjnr207p6977hr4bsnczryg609rh";
   };
 
+  buildInputs = [ wofi wtype wl-clipboard ];
   nativeBuildInputs = [ jq ];
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/wofi-emoji/default.nix:20]:
```suggestion
  nativeBuildInputs = [ jq ];
  buildInputs = [ wofi wtype wl-clipboard ];
```

File: pkgs/applications/misc/wofi-emoji/default.nix [hunk 2]
```diff
@@ -1,30 +1,27 @@
...
 
   postPatch = ''
-    cp "${emojiJSON}/db/emoji.json" .
     substituteInPlace build.sh \
-      --replace 'curl https://raw.githubusercontent.com/github/gemoji/master/db/emoji.json' 'cat emoji.json'
+      --replace 'curl https://raw.githubusercontent.com/muan/emojilib/v3.0.6/dist/emoji-en-US.json' 'cat emoji-en-US.json'
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/wofi-emoji/default.nix:24]:
```suggestion
      --replace 'curl ${emojiJSON.url}' 'cat emoji-en-US.json'
```

Comment (riceicetea) [reply pkgs/applications/misc/wofi-emoji/default.nix:24]: True, thanks.

File: pkgs/applications/misc/wofi-emoji/default.nix (hunk 1 above)
Comment (riceicetea) [reply pkgs/applications/misc/wofi-emoji/default.nix]: Is this just formatting sugar, or does this have a purpose?

Comment (SuperSandro2000) [reply pkgs/applications/misc/wofi-emoji/default.nix]: just formatting

File: pkgs/applications/misc/wofi-emoji/default.nix [hunk 3]
```diff
@@ -1,31 +1,28 @@
...
   nativeBuildInputs = [ jq ];
+  buildInputs = [ wofi wtype wl-clipboard ];
 
   postPatch = ''
-    cp "${emojiJSON}/db/emoji.json" .
     substituteInPlace build.sh \
-      --replace 'curl https://raw.githubusercontent.com/github/gemoji/master/db/emoji.json' 'cat emoji.json'
-  '';
+      --replace 'curl ${emojiJSON.url}' 'cat emoji-en-US.json'
+      '';
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/wofi-emoji/default.nix:25]:
```suggestion
  '';
```

**Pull request merged**